frontend/src/metabase/components/AccordionList.jsx

```diff
@@ -251,7 +262,9 @@ export default class AccordionList extends Component {
     let searchFilter = () => true;
     if (searchText) {
       searchFilter = item => {
-        let itemText = String(item[searchProp] || "");
+        let itemText = searchProp.includes(".")
+          ? this.getInItem(item, searchProp)
+          : String(item[searchProp] || "");
```

Comment:
We can use [`getIn` from `icepick`](https://github.com/aearly/icepick#getincollection-path) here:

```js
const path = searchProp.split(".")
let itemText = String(getIn(item, path) || "")
```

Would work for nested and not nested paths